src/AuthenticatedAPIClient/tests/AuthenticatedAPIClient.test.jsx

```diff
@@ -100,6 +105,11 @@ function expectLogin(redirectUrl = process.env.BASE_URL) {
     .toHaveBeenCalledWith(`${process.env.LOGIN_URL}?next=${encodedRedirectUrl}`);
 }
 
+const expectLogErrorToHaveBeenCalledWithMessage = (message) => {
+  const loggedError = mockLoggingService.logError.mock.calls[0][0];
```

Comment:
What about checking that the right number of calls happened, or the messages of other calls?

Reply:
Done